tests/test_workspace_repowise.py

```
@@ -0,0 +1,29 @@
+def test_get_context_integration(tools):
+    context = tools.get_context(["app.py"])
+    assert "<file path=\"app.py\">" in context
+
+def test_get_risk_integration(tools):
+    risk = tools.get_risk()
+    assert "overall_hotspots" in risk
+
+def test_get_why_integration(tools, tmp_path):
+    # Just check it returns a string without crashing
+    import subprocess
+    subprocess.run(["git", "init"], cwd=tmp_path, check=True)
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)